LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/Payments/Invoice.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.Payments
+{
+    public class Invoice
+    {
+        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public Guid Id { get; set; }
```

Comment:
Can we have invoice Id as long this time? There were complaints on the guid last time. More secure but because it's shown on the design, looks terrible 
![image](https://user-images.githubusercontent.com/81692554/139876372-1969a231-52e6-46b9-bdd8-d8cc926c6e7a.png)

Also invoice number column maybe? (string)

Reply:
Agreed Duncan. I would prefer to use Invoice number column as unique.

Reply:
@brkzkn, do you agree?

Reply:
![image](https://user-images.githubusercontent.com/81692554/139901433-35ae2950-c6ae-4699-b6d4-5e639dace690.png)
We will need that invoice number anyway. If we can convince designer to use that on FE instead of Id which is a Guid then no need to change type of Id

Reply:
added unique invoice number

will merge for now, the rest can be updated in ongoing mode